lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -31,6 +31,9 @@
 CONFIG_OVERRIDE_DIR = "/var/opt/magma/tmp"
```

Comment:
Couple things with this file

(a) The `_get_xxx(service_config...)` pattern seems to be choosing between a value from the passed-in mconfig vs. some default value
1.  The `service_config` arg should be renamed to `service_mconfig` (right?), which delineates whether that config value is coming from mconfig (local, with orc8r override) vs. service config (local)
2. In some cases, we're prioritizing the value from the mconfig (e.g. in `_get_apn_correction_map_list`), and in others we're prioritizing the value from the service config (e.g. in `_get_enable_nat`). If this behavior is unintentional, we should have a clear+consistent priority for all getters (and I assume we want to prioritize mconfig, but that's up to you). If this behavior *is* intentional, we should clearly document (e.g. in the relevant function docstrings) why the particular priority on mconfig vs. service config is made for that function